packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.0.0
+
+* Bumps androidx.webkit:webkit from 1.12.0 to 1.12.1.
+* **Breaking Change** Bumps Android `minSdkVersion` from 19 to 21.
```

Comment:
I thought we were going to do this as a minor version because this isn't actually breaking, since Flutter 3.24 requires 21+, so nobody should be able to be broken by this.

Reply:
Ah I thought that only applied to `webview_flutter`. And we would make this a breaking change for the plugin.

Reply:
Sorry I should have gotten an additional review for clarification. Will this bump cause conflicts and do we need to revert the version?

Reply:
It's going to make things unnecessarily difficult for anyone doing https://pub.dev/packages/webview_flutter#platform-specific-features because they will get resolver errors the next time they try to update `webview_flutter`, and will have to figure out they need to update their `webview_flutter_android` major version.

Withdrawing the published version would create a problem for us later when we actually want to bump the major version again, so I think we'll just have to live with the added friction for people with dependencies on the platform version.

Reply:
Ah ok now I understand. Could we change the `webview_flutter` dependency to `>=3.16.0 <5.0.0` temporarily? Or are we better off just setting it to ^4.0.0 with the friction.

Reply:
That would just kick the can down the road very slightly, because the next time any feature is added we'll have to update the minimum endorsed versions to pick up the feature. Better to just get it out of the way now, rather than make the next feature update slightly more complex.